quoridor-app/app/component/__tests__/square-test.js

```diff
@@ -6,7 +6,7 @@ import Square from "../square";
 describe("Square Component tests", () => {
   it("should renders the right component", () => {
     const tree = renderer
-      .create(<Square position="{top: 0, left: 10}" />)
+      .create(<Square position="{top: 0, left: 10}" disable={true} />)
```

Comment:
You don't need to add `={true}` in JSX for the booleans attributes. It's the same as:
``` <Square position="{top: 0, left: 10}" disable />```